composer.json

```diff
@@ -33,7 +33,6 @@
         "ext-sqlite3": "*",
         "dg/bypass-finals": "^1.3",
         "doctrine/doctrine-bundle": "^2.7",
-        "doctrine/mongodb-odm": "^2.4",
```

Comment:
this change is unrelated and should be reverted

Reply:
Totally, i removed this package by mistake and forgot to put it back... Thanks ! 👍🏼